apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/TransformerUtils.java

```diff
@@ -1549,7 +1549,8 @@ static void addCommonEobInformationInpatientSNF(
        */
       eob.addDiagnosis()
           .setPackageCode(
-              createCodeableConcept(eob, CcwCodebookVariable.CLM_DRG_CD, diagnosisRelatedGroupCd));
+              createCodeableConcept(eob, CcwCodebookVariable.CLM_DRG_CD, diagnosisRelatedGroupCd))
+          .setSequence(eob.getDiagnosis().size());
```

Comment:
Why is the sequence the size of the diagnosis here? We can't be doing every sequence like this (or theyd all be at equal and no longer a sequence); how are others set? I figured this would be some deterministic value that would take into account other items in the sequence.

Reply:
done